### PR TITLE
[main-blast][feat] Integrate ybTKNs to HMX on Blast

### PR DESCRIPTION
**Pull request by shutsuwei**

### CODE REVIEW

File: src/handlers/CrossMarginHandler.sol [hunk 1]
```diff
@@ -408,26 +428,50 @@ contract CrossMarginHandler is OwnableUpgradeable, ReentrancyGuardUpgradeable, I
...
+        _order.account,
+        _order.subAccountId,
+        _order.token,
+        _order.amount,
+        address(this)
+      );
+      // Then we redeem ybs.
+      uint256 _assets = IYBToken(_order.token).redeem(_order.amount, address(this), address(this));
+      // Transfer to user
+      ERC20Upgradeable(_originalToken).safeTransfer(_order.account, _assets);
```
Comment (john-xina-p88) [inline src/handlers/CrossMarginHandler.sol:474]: For gas saving, we can shorten this to
```suggestion
      uint256 _assets = IYBToken(_order.token).redeem(_order.amount,order.account, address(this));
```

File: src/storages/ConfigStorage.sol [hunk 2]
```diff
@@ -653,6 +655,21 @@ contract ConfigStorage is IConfigStorage, OwnableUpgradeable {
...
+  function setYbTokenOfMany(address[] calldata _tokens, address[] calldata _ybTokens) external onlyOwner {
+    if (_tokens.length != _ybTokens.length) revert IConfigStorage_BadArgs();
+
+    for (uint256 i = 0; i < _tokens.length; ) {
+      ybTokenOf[_tokens[i]] = _ybTokens[i];
+
+      unchecked {
+        ++i;
+      }
+    }
```
Comment (kim-hera) [inline src/storages/ConfigStorage.sol:670]: emit event

File: src/storages/ConfigStorage.sol [hunk 3]
```diff
@@ -653,6 +655,21 @@ contract ConfigStorage is IConfigStorage, OwnableUpgradeable {
     }
   }
 
+  /// @notice Set ybToken for a token
+  /// @param _tokens The token address to set ybToken
+  /// @param _ybTokens The ybToken address to set
+  function setYbTokenOfMany(address[] calldata _tokens, address[] calldata _ybTokens) external onlyOwner {
+    if (_tokens.length != _ybTokens.length) revert IConfigStorage_BadArgs();
+
+    for (uint256 i = 0; i < _tokens.length; ) {
```
Comment (kim-hera) [inline src/storages/ConfigStorage.sol:664]: Gas saving
```suggestion
    uint tokenLength = _tokens.length;
    if (tokenLenght != _ybTokens.length) revert IConfigStorage_BadArgs();

    for (uint256 i = 0; i < tokenLength; ) {
```
